src/main/java/org/cru/redegg/reporting/InMemoryErrorQueue.java

```diff
@@ -82,6 +93,22 @@ public void shutdown()
     @Override
     public void enqueue(final ErrorReport report)
     {
+        try
+        {
+            submit(report);
+        }
+        catch (RejectedExecutionException e)
+        {
+            System.out.println("submission rejected");
```

Comment:
Oops! Sloppy me. I had those in to debug a problem I had with Mockito in the unit test. (The test would only pass sporadically. I was using `@InjectMocks`, but it turns out that since two of the constructor parameters have the same type, Mockito only wired up the right ones half the time.)

I'll remove them.